PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+<!--- Explain your issue / bug / feature -->
+
+## Checklist
+
+- [ ] Tests added if applicable
+- [ ] README updated if applicable
+- [ ] CHANGELOG updated if applicable
```

Comment:
Should we add: "Breaking changes marked in the commit message"? See [CONTRIBUTING.MD](https://github.com/coredump-ch/spaceapi-rs/blob/1db21c76fa73566c89aa57963dfefee66fd0fc01/CONTRIBUTING.md)